docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -143,6 +143,48 @@ To send a command from the client and to process it on the server side, follow t
     }
     ```
 
+To send a search command from the client and to process it on the server side, follow these steps:
```

Comment:
I have reviewed the modified part, I have no comments, and there are no changes required in the modified part.

@hsgwon , I just have one suggestion, if this change is in your scope, please make this change.

Line number 10:

You can [create a playlist](#updating-and-retrieving-playlist) on the server side, and then retrieve that information on the client side.

In this line, 'create a playlist' is hyperlinked to 'Updating and Retrieving Playlist'. It would be better to change the heading from 'Updating and Retrieving Playlist' to 'Creating and Retrieving Playlist' to maintain consistency with other hyperlinks.

If this is not under your scope, please ignore my comments.

Reply:
Thank you for your review. I'll fix it.